### ISSUE TITLE
syntax error, unexpected ')' in phpfileuploading/src/Upload.php

### ISSUE THREAD

**Issue by karamellpelle** [farisc0de/Uploady#13]
I'm trying to install your code on my server, but it does not work. I've uploaded your files (and set 777 on _uploads/_, however I forgot that during my first run), and I get redirected to _/install.php_ in my browser. But this is just a blank page.

There is an _error_log_ file in the server directory with `[08-Nov-2023 14:28:56 UTC] PHP Parse error:  syntax error, unexpected ')' in /home/xxx/public_html/yyy/vendor/farisc0de/phpfileuploading/src/Upload.php on line 222` .

I can't file see any wrong with parenthesis in this file. What could the problem be?

Your project looks great, so I want it up and running :)

**Comment by farisc0de** [comment]
It is a weird issue as there is no syntax error in the code.

Please remove everything and try to install the latest version and let me know what happens.

**Comment by karamellpelle** [comment]
Yes, this is weird. I could not find a syntax error in that file neither. Could it be something about strings and escape characters in the domain name/etc. ? My domain has a `3` in it, but I don't think that should be a problem. I will now test the project code on a local raspberry pi server!

I did a fresh install with newest commit on a subdomain of my web hotel.

BTW, I copied the content in the _uploady/_ folder (.php, directories etc) into my subdomain's folder of my server, not the project folder (where _README.md_ is located). I guessed that was correct.

I did create a MySQL database and added it to the _config.php_, but the database is still empty.

The code is running since it loads the _install.php_ file, but then it fails somehow.

Thanks!
